### DISCUSSION TITLE
SSL Exception

### DISCUSSION THREAD

**Discussion by jparangdev** [getsentry/sentry-java#2589] [Q&A] [answered]
Hi team, 

May I know why ssl exception occurs? It works well local environments, but not in DEV or PROD environments.

What I'm curious about is that it works well on other servers with the same network settings(same AWS security group)

The difference between the server and the server that works well is the sentry version and the java version.

A - error
B - well

A - openjdk11, springboot 2.6, sentry 6.11
B - java 1.8, springboot 2.3, sentry 6.6

<img width="948" alt="스크린샷 2023-02-28 오후 5 28 42" src="https://user-images.githubusercontent.com/75261925/223088337-fbd71a00-0da9-47ab-93b9-76184c754c6a.png">

**Comment by adinauer** [comment, marked as answer]
Hello @jparangdev have you tried using curl to access Sentry? Maybe your CA certificates are outdated.

**Reply by jparangdev** [reply]
Which end point should I run curl?
 I've tried with DSN address both A and B servers, But I think it is not work..
 Attach the log.

<img width="742" alt="A" src="https://user-images.githubusercontent.com/75261925/223299565-ab76672a-7350-43ac-a88a-94f6bab7a58d.png">
<img width="860" alt="B" src="https://user-images.githubusercontent.com/75261925/223299570-82e43f14-daeb-44b2-9735-802b88ec5378.png">

**Reply by jparangdev** [reply]
It was a firewall issue. Thanks for the help.

**Reply by adinauer** [reply]
@jparangdev glad to hear you figured it out.
